Dockerfile-base

```diff
@@ -1,41 +1,63 @@
+# --------------------------------------------------
 # Sharelatex Base Image (sharelatex/sharelatex-base)
+# --------------------------------------------------
 
-FROM phusion/baseimage:0.9.16
+FROM phusion/baseimage:0.11
 
 ENV baseDir .
 
+
+# Install dependencies
+# --------------------
 RUN apt-get update
-RUN curl -sL https://deb.nodesource.com/setup_6.x | sudo bash -
-RUN apt-get install -y build-essential wget nodejs unzip time imagemagick optipng strace nginx git python zlib1g-dev libpcre3-dev aspell aspell-en aspell-af aspell-am aspell-ar aspell-ar-large aspell-bg aspell-bn aspell-br aspell-ca aspell-cs aspell-cy aspell-da aspell-de aspell-de-alt aspell-el aspell-eo aspell-es aspell-et aspell-eu-es aspell-fa aspell-fo aspell-fr aspell-ga aspell-gl-minimos aspell-gu aspell-he aspell-hi aspell-hr aspell-hsb aspell-hu aspell-hy aspell-id aspell-is aspell-it aspell-kk aspell-kn aspell-ku aspell-lt aspell-lv aspell-ml aspell-mr aspell-nl aspell-no aspell-nr aspell-ns aspell-or aspell-pa aspell-pl aspell-pt-br aspell-ro aspell-ru aspell-sk aspell-sl aspell-ss aspell-st aspell-sv aspell-ta aspell-te aspell-tl aspell-tn aspell-ts aspell-uk aspell-uz aspell-xh aspell-zu 
+RUN apt-get install -y sudo
+RUN apt-get install -y build-essential wget net-tools unzip time imagemagick optipng strace nginx git python zlib1g-dev libpcre3-dev aspell aspell-en aspell-af aspell-am aspell-ar aspell-ar-large aspell-bg aspell-bn aspell-br aspell-ca aspell-cs aspell-cy aspell-da aspell-de aspell-el aspell-eo aspell-es aspell-et aspell-eu-es aspell-fa aspell-fo aspell-fr aspell-ga aspell-gl-minimos aspell-gu aspell-he aspell-hi aspell-hr aspell-hsb aspell-hu aspell-hy aspell-id aspell-is aspell-it aspell-kk aspell-kn aspell-ku aspell-lt aspell-lv aspell-ml aspell-mr aspell-nl aspell-nr aspell-ns  aspell-pa aspell-pl aspell-pt aspell-pt-br aspell-ro aspell-ru aspell-sk aspell-sl aspell-ss aspell-st aspell-sv aspell-tl aspell-tn aspell-ts aspell-uk aspell-uz aspell-xh aspell-zu
+
 
+# Install qpdf
+# ------------
 WORKDIR /opt
 RUN wget https://s3.amazonaws.com/sharelatex-random-files/qpdf-6.0.0.tar.gz && tar xzf qpdf-6.0.0.tar.gz
 WORKDIR /opt/qpdf-6.0.0
 RUN ./configure && make && make install && ldconfig
 
+
+# Install Node
+# ------------
+RUN curl -sL https://deb.nodesource.com/setup_10.x | sudo bash -
+RUN apt-get install -y nodejs
+RUN npm install -g grunt-cli
+
+
+# Install Node6 (required by some services)
+# -----------------------------------------
+RUN wget https://nodejs.org/dist/v6.17.1/node-v6.17.1-linux-x64.tar.gz && \
+	mkdir -p /opt/nodejs && tar -xzf node-v6.17.1-linux-x64.tar.gz -C /opt/nodejs/ && \
+    cd /opt/nodejs && mv node-v6.17.1-linux-x64 6.17.1 && \
+    ln -s /opt/nodejs/6.17.1/bin/node /usr/bin/node6
+
+
 # Install TexLive
-RUN wget http://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz; \
+# ---------------
+RUN wget https://ctan.crest.fr/tex-archive/systems/texlive/tlnet/install-tl-unx.tar.gz; \
```

Comment:
I think this link should still work if we keep the `mirror.ctan.org` hostname --- at the moment this is hard coded to one CTAN mirror, which may not be the best one for the user's location. Also, the original link seems to be working OK for me; is there a particular reason to change it?

Also, it's not a new issue, but while we're here: I think it's better to use `&&` than `;` in these commands, so a failure in e.g. the `wget` step will stop the build right away, rather than continuing to the next command in the sequence.

Reply:
1. Mirror: that's correct. I updated it locally since my closest mirrors had problems and forgot to revert it.

2. Will use `&&` here and in other multiline `RUN`.